File: GreenployeeClient/src/app/model/meta.ts
import { BaseClass } from "./baseClass";
import { PessoaMeta } from "./pessoaMeta";

export class Meta extends BaseClass {

    dsRecompensa! : string;
    dtInicio! : Date;
    dtFim! : Date;
    vlMeta! : number;
    pessoasMeta!: PessoaMeta [];

    constructor() {
        super();
        this.pessoasMeta = [];
    }  
}

File: Greenployee.MODELS/Model/Meta.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Greenployee.MODELS.Model
{
    public class Meta : BaseClass
    {
        [StringLength(255)]
        public string dsRecompensa { get; set; } = string.Empty;

        public DateTime dtInicio { get; set; }

        public DateTime dtFim { get; set; }

        public decimal? vlMeta { get; set; }

        [InverseProperty("Meta")]
        public virtual ICollection<PessoaMeta> PessoasMeta { get; set; }
    }

}